test/specs/common/tests/02.column.js

```diff
@@ -0,0 +1,194 @@
+exports.execute = function(options) {
+    var sampleValues = {
+        date: '2016-05-02',
+        timestamptz: '2011-05-06T08:25:25-07:00'
+    };
+
+    describe('About the Column class, ', function() {
+        var schemaName1 = "common_schema_1",
+            schemaName2 = "common_schema_2";
+        var table1_schema1, // has two outbound fks to table2_schema1. with foreign-key annotation.
```

Comment:
Since you're just using one table (table1_schema2), you don't need this variable. The same goes for `schemaName1`, `table1_schema1`, and `table2_schema1.`